### ISSUE TITLE
Text size changes on "detail" scene

### ISSUE THREAD

**Issue by mflint** [mflint/ios-tldr-viewer#8]
```
Small problem in Slideover:

- open Tldr, view a man page
- open other app
- open slideover and select Tldr 
=> your app opens and shows the previous man page, but the font has the same size as in fullscreen mode. You have to zoom to fit the man-page manually
```

(Reported by Michael Schmidt via email)

**Comment by mflint** [comment]
The best way to fix this might be to convert Markdown to NSAttributedString, not HTML.

This fills me with dread.

**Comment by hacknicity** [comment]
FWIW, it's very easy to convert HTML to NSAttributedString:

```
NSString *string = /* some HTML */;

NSMutableAttributedString *attributedString= [[NSMutableAttributedString alloc] initWithData:[string dataUsingEncoding:NSUTF8StringEncoding]
                                                                                     options:@{NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType}
                                                                          documentAttributes:nil
                                                                                       error:nil];
```

**Comment by mflint** [comment]
WebView zoom level fixed in v1.0.2
